config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -86,6 +86,59 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-node-e2e-kubetest2
+    # explicitly needs /test pull-kubernetes-node-e2e to run
+    always_run: false
+    # if it is run and fails, don't block the PR
+    optional: true
+    branches:
+    - master
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
```

Comment:
we can drop the need to check out the test-infra repo after we add the raw url functionality mentioned in the other comment.

let's start with whatever we have for now.